src/integration/java/io/pinecone/integration/controlPlane/pod/ConfigureIndexTest.java

```diff
@@ -20,16 +20,12 @@
 
 public class ConfigureIndexTest {
     private static PineconeControlPlaneClient controlPlaneClient;
-    private String indexName;
+    private static String indexName;
     private static final Logger logger = LoggerFactory.getLogger(PineconeClientLiveIntegTest.class);
 
     @BeforeAll
-    public static void defineControlPlaneClient() {
+    public static void setUp() throws InterruptedException, IOException {
         controlPlaneClient = new PineconeControlPlaneClient(System.getenv("PINECONE_API_KEY"));
-    }
-
-    @BeforeEach
```

Comment:
This didn't seem like it needed to be called before each test, I think doing it before everything is fine, but let me know if not.

Reply:
The reason we want this is to make sure each test starts by checking if the starting state is correct. We already have a condition in each of the tests and previously, we used to have an extra test which was removed, so it should safe to remove this.